docs/ref-info/ref-tokens.md

```diff
@@ -0,0 +1,56 @@
+---
+title: UID2 Tokens and Refresh Tokens
+description: Information for publishers about UID2 tokens and refresh tokens.
+hide_table_of_contents: false
+sidebar_position: 06
+---
+
+import Link from '@docusaurus/Link';
+
+# UID2 Tokens and Refresh Tokens
+
+When a publisher sends a user's <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> (email address or phone number) to the UID2 Operator, whether via one of the UID2 SDKs or the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) endpoint, the UID2 Operator returns a <a href="glossary-uid#gl-uid2-token">UID2 token</a> with associated values. The token is an opaque alphanumeric string, and is pseudonymous. Because of the way UID2 works, different instances of activity, on browsers, CTV, and electronic devices such as phone and tablets, can be matched without compromising the privacy of the individual. The token is designed so that it cannot be reverse engineered to arrive at the original email address or phone number.
+
+For security reasons, the UID2 token has a short life. Along with the UID2 token, the UID2 Operator sends a <a href="glossary-uid#gl-refresh-token">refresh token</a> that the publisher can use to generate a new UID2 token. If the token is not refreshed before it expires, it becomes invalid and cannot be used for targeted advertising.
+
+## UID2 Tokens: Key Information
+
+Here are some key points about UID2 tokens:
+
+- The UID2 token is a unique value: no two UID2 tokens are the same.
+- UID2 tokens are case sensitive.
+- The token value is opaque: do not make any assumptions about the format or about the length of the string.
+- The token has a limited life, but can be refreshed using the refresh token.
+- The token can be refreshed many times, to get a new UID2 token, as long as the UID2 token is always refreshed before the refresh token expires.
+- If the token has expired, or as an alternative to refreshing an existing token, you can always generate a new UID2 token from the original hashed or unhashed email address or phone number.
+- Publishers send UID2 tokens in the bidstream.
+- Refreshing a UID2 token does not invalidate/expire the original or previous UID2 token. You can still use the earlier token until it expires.
+- When the UID2 Operator service receives the refresh token with a request for a new UID2 token, it checks for user opt-out. If the user has opted out of UID2, no new UID2 token is generated. For details, see [User Opt-Out](../getting-started/gs-opt-out.md).
```

Comment:
this line should also be copied and pasted in refresh token section below

Maybe in UID2 token section here, we say token generation logic will also check for user optout instead?

Reply:
Done on both... great edits, thx.